<launch>
    <arg name="arg_initial_yaw" value="90"/>
    <param name="/use_sim_time" value="true"/>
    <param name="/need_init_odom" value="true"/>
    <param name="/need_init_map" value="true"/> 
    <node pkg="chassis_navigation" type="gnss_odom_node" name="gnss_odom_node">
        <param name="need_calibration" value="true"/>
        <param name="gnss_rate" value="1.0"/>
        <param name="v_min" value="0.5"/>
        <param name="debug_alone" value="true"/>
        <param name="initial_yaw" value="$(arg arg_initial_yaw)"/>
        <!-- coli [0.9 ] -->
        <param name="coli" value="1.0"/>
        <!-- coan [0.7 0.75] -->
        <param name="coan" value="0.72"/>
    </node>
    <arg name="map" default="emptymap.yaml" />
    <node name="map_server" pkg="map_server" type="map_server" args="$(find chassis_navigation)/map/$(arg map)"/>
    <node pkg="tf" type="static_transform_publisher" name="map_to_odom" args="0 0 0 1.57 0 0 /map /odom 20" />
    <node pkg="tf" type="static_transform_publisher" name="map_to_gnssOdom" args="0 0 0 0 0 0 /map /gnssOdom 20" />
    <node pkg="rviz" type="rviz" name="rviz" args="-d $(find chassis_navigation)/rviz/debug_gnssOdom.rviz"/>
</launch>